lib/mp4/transmuxer.js

```diff
@@ -738,12 +792,20 @@ CoalesceStream = function(options, metadataStream) {
   this.numberOfTracks = 0;
   this.metadataStream = metadataStream;
 
+  options = options || {};
+
   if (typeof options.remux !== 'undefined') {
     this.remuxTracks = !!options.remux;
   } else {
     this.remuxTracks = true;
   }
 
+  if (typeof options.keepOriginalTimestamps === 'boolean') {
+    this.keepOriginalTimestamps = options.keepOriginalTimestamps;
+  } else {
+    this.keepOriginalTimestamps = false;
```

Comment:
Actually, it seems that it's always a boolean (re: comment above) since we default here?

Also, can change to: `this.keepOriginalTimestamps = !!(options.keepOriginalTimestamps);` or rely on truthiness without explicitly making it a boolean.

Reply:
I figured we might want to only set the value if a boolean is passed in (rather than a string or some other truthy value.) Video.js core does this kind of thing for safety, so was using that pattern.